src/System.Runtime.WindowsRuntime/ref/Configurations.props

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <PackageConfigurations>
+      netcoreapp;
```

Comment:
Are you trying to avoid having a uap asset in the package? The general pattern is:
```
<PackageConfigurations>
  netcoreapp;
</PackageConfigurations>
<BuildConfigurations>
   $(PackageConfigurations);
  uap
</BuildConfigurations>

```

Reply:
Fixed